parts/linux/cloud-init/artifacts/health-monitor.sh

```diff
@@ -71,7 +76,12 @@ component=$1
 echo "Start kubernetes health monitoring for ${component}"
 
 if [[ "${component}" == "container-runtime" ]]; then
-  container_runtime_monitoring
+  if [[ -z $2 ]]; then
+    echo "Usage: health-monitor.sh container-runtime <docker/containerd>"
+    exit 1
+  fi
```

Comment:
consider moving the validation to the beginning and stay together with the other validation

Reply:
addressed as per comment belwo